loops_lab.js

```diff
@@ -0,0 +1,222 @@
+// // ## Exercises
+// //
+// // 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
```

Comment:
Generally good understanding of for and while loops. Attempt some bonuses questions in next assignments. Keep up the good work!